stl/inc/xutility

```diff
@@ -4963,35 +4958,80 @@ _NODISCARD bool lexicographical_compare(
 #endif // _HAS_CXX17
 
 #ifdef __cpp_lib_concepts
+template <class _Elem1, class _Elem2, class _Cmp>
+struct _Lex_compare_three_way_memcmp_classify_comp {
+    using _Comp = void;
+};
+
+template <class _Elem1, class _Elem2>
+struct _Lex_compare_three_way_memcmp_classify_comp<_Elem1, _Elem2, compare_three_way> {
+    using _Comp = conditional_t<_Lex_compare_memcmp_classify_elements<_Elem1,
+                                    _Elem2> && three_way_comparable_with<const _Elem1&, const _Elem2&>,
+        compare_three_way, void>;
+};
+
+template <class _Ty1, class _Ty2>
+concept _Can_strong_order = requires(const _Ty1& _Left, const _Ty2& _Right) {
+    strong_order(_Left, _Right);

```

Comment:
Nitpick: Although this technically isn't a function (and therefore is invulnerable to Argument-Dependent Lookup), we've conventionally qualified it elsewhere just to avoid any confusion. Occurs below for `weak_order` and `partial_order`. Example precedent:

https://github.com/microsoft/STL/blob/d5feb03f320ccaa8e214e3010c40c9c5afd686e2/stl/inc/compare#L700